.github/workflows/main.yml

```
@@ -40,6 +40,8 @@ jobs:
         run: black --check --diff retworkx rustworkx retworkx tests
       - name: Python Lint
         run: flake8 --per-file-ignores='retworkx/__init__.py:F405,F403' setup.py retworkx tests rustworkx
+      - name: Check stray release notes
+        run: python3 tools/find_stray_release_notes.py
```

```suggestion
        run: python tools/find_stray_release_notes.py
```

It is implied we use Python 3 by the other commands